list.json

```diff
@@ -2175,13 +2170,6 @@
     "name": "Webhook",
     "github": "webhook/webhook"
   },
-  {
-    "name": "Website Meta Language",
```

Comment:
The project website is not available anymore.

Reply:
Considering that the last release was in 2006, I assume the project doesn't exist anymore.